ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.md

```diff
@@ -0,0 +1,790 @@
+
```

Comment:
This is auto generated by validate state and we have not touched this in this PR - I just removed it but this will impact existing user of eos_validate_state